<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jacqueline Houtman">
<meta name="author" content="Rebecca Glassman">
<meta name="author" content="Lindsey Shultz">
<meta name="author" content="Jessica Malaty Rivera">
<meta name="author" content="Emily Bass">
<meta name="author" content="Rick Bright">
<meta name="dcterms.date" content="2021-12-01">

<title>PTC - Genomic Surveillance Is Essential To Track Covid-19 Variants in Both Unvaccinated and Vaccinated Populations</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

<script type="text/javascript">

(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-234526530-1', { 'storage': 'none' });

ga('send', {
  hitType: 'pageview',
  'anonymizeIp': true,
});
</script>


</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../ptc-bars.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">PTC</span>
    </a>
  </div>
          <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html">
 <span class="menu-text">Anthology</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../scorecards.html">
 <span class="menu-text">Scorecards</span></a>
  </li>  
</ul>
          <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="../../about.html"><i class="bi bi-info-circle-fill" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/pandemictrack"><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents</h2>
   
  <ul>
  <li><a href="#sequencing-detects-variants." id="toc-sequencing-detects-variants." class="nav-link active" data-scroll-target="#sequencing-detects-variants.">Sequencing detects variants.</a></li>
  <li><a href="#genomic-surveillance-can-help-map-the-pandemic-and-detect-changes." id="toc-genomic-surveillance-can-help-map-the-pandemic-and-detect-changes." class="nav-link" data-scroll-target="#genomic-surveillance-can-help-map-the-pandemic-and-detect-changes.">Genomic surveillance can help map the pandemic and detect changes.</a></li>
  <li><a href="#recent-advances-in-technology-have-made-sequencing-quicker-and-more-affordable." id="toc-recent-advances-in-technology-have-made-sequencing-quicker-and-more-affordable." class="nav-link" data-scroll-target="#recent-advances-in-technology-have-made-sequencing-quicker-and-more-affordable.">Recent advances in technology have made sequencing quicker and more affordable.</a></li>
  <li><a href="#obtaining-viral-sequences-from-underrepresented-populations-requires-more-than-just-technology" id="toc-obtaining-viral-sequences-from-underrepresented-populations-requires-more-than-just-technology" class="nav-link" data-scroll-target="#obtaining-viral-sequences-from-underrepresented-populations-requires-more-than-just-technology">Obtaining viral sequences from underrepresented populations requires more than just technology,</a></li>
  <li><a href="#genomic-surveillance-is-made-possible-through-collaboration-outreach-and-trust." id="toc-genomic-surveillance-is-made-possible-through-collaboration-outreach-and-trust." class="nav-link" data-scroll-target="#genomic-surveillance-is-made-possible-through-collaboration-outreach-and-trust.">Genomic surveillance is made possible through collaboration, outreach, and trust.</a></li>
  <li><a href="#engaging-the-community-also-means-empowering-their-clientele-with-information." id="toc-engaging-the-community-also-means-empowering-their-clientele-with-information." class="nav-link" data-scroll-target="#engaging-the-community-also-means-empowering-their-clientele-with-information.">Engaging the community also means empowering their clientele with information.</a></li>
  </ul>
</nav>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Genomic Surveillance Is Essential To Track Covid-19 Variants in Both Unvaccinated and Vaccinated Populations</h1>
<p class="subtitle lead">Technological advancements have made sequencing quicker and more affordable, but collaboration, outreach, and trust are needed to achieve representative samples from underserved populations.</p>
  <div class="quarto-categories">
    <div class="quarto-category">Variants</div>
  </div>
  </div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Jacqueline Houtman </p>
             <p>Rebecca Glassman </p>
             <p>Lindsey Shultz </p>
             <p>Jessica Malaty Rivera </p>
             <p>Emily Bass </p>
             <p>Rick Bright </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">December 1, 2021</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>This piece is originally published at <a href="https://www.rockefellerfoundation.org/blog/genomic-surveillance-is-essential-to-track-covid-19-variants-in-both-unvaccinated-and-vaccinated-populations/">rockefellerfoundation.org/blog/genomic-surveillance-is-essential-to-track-covid-19-variants-in-both-unvaccinated-and-vaccinated-populations/</a></p>
</div>
</div>
<p>The course of the Covid-19 pandemic depends not only on how the SARS-CoV-2 virus changes, but on how communities respond to it. Public health officials can only make decisions to protect their communities if they know how much virus is circulating locally, which <a href="https://www.rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/">viral variants</a> are present, and whether variants are emerging that may be more virulent or transmissible, or less susceptible to vaccines or medications. The only way to do this is through adequate genomic sequencing, which may not be available in communities that need it. Improvements in sequencing technology, training, and lab availability can help overcome barriers to representation and strengthen sequencing capacity and fill in large gaps in genomic data in the US and around the world. Collaboration and trust are crucial for working with communities to track the virus and get ahead of the pandemic.</p>
<section id="sequencing-detects-variants." class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="sequencing-detects-variants.">Sequencing detects variants.</h3>
<p>A <strong>diagnostic PCR test</strong> can tell whether SARS-CoV-2 is present in a sample–but <a href="https://www.krisp.org.za/manuscripts/25Nov2021_B.1.1.529_Media.pdf">in most cases</a> not which <a href="https://www.rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/">variant</a>. This is because the test detects limited, preselected portions of viral genes. In contrast, <strong>whole genome sequencing</strong> reveals the entirety of the SARS-CoV-2’s genetic material, also known as its genome. Once a full genome is sequenced, scientists can share data globally and compare it to an array of other genomes collected in global databases such as <a href="https://www.gisaid.org/">GISAID</a>. Comparative analysis lets them identify mutations–or changes–from the original (wild type) virus <a href="https://www.who.int/news/item/27-04-2020-who-timeline---covid-19">sequenced and shared with the global GISAID community in January 2020</a>. This analysis can spot an accumulation of mutations that could change how a variant virus behaves. Changes like increased transmissibility, an altered clinical course, decreased effectiveness of therapeutic treatments, or <a href="https://www.nature.com/articles/s41423-021-00779-5">vaccine escape</a>may lead to designation as <a href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">Variants of Interest (VOI) or Variants of Concern (VOC)</a>, which are given Greek letter names by the <a href="https://covid19.who.int/">World Health Organization (WHO)</a>. While the <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/delta-variant.html">Delta variant</a> continues to be the <a href="https://outbreak.info/situation-reports/delta?loc=USA&amp;loc=USA_US-CA&amp;selected=Worldwide&amp;overlay=false">dominant lineage worldwide,</a>the WHO designated the Omicron variant as a new VOC on <a href="https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern">November 26</a>.</p>
<p>Extensive laboratory studies can help determine the role of specific mutations, which can help predict the hallmarks of newly identified variants. The table below lists some <a href="https://www.nytimes.com/interactive/2021/health/coronavirus-variant-tracker.html#D614G">important mutations</a> in the viral spike protein that have been shown to affect the characteristics of the five WHO-designated VOCs, including <a href="https://www.ecdc.europa.eu/en/covid-19/variants-concern">Omicron</a>. Mutations in <a href="https://www.the-scientist.com/news-opinion/tweak-to-n-protein-makes-delta-variant-more-infectious-69386">other proteins</a> may also affect the properties of viral variants.</p>
<div id="tbl-1" class="tbl-parent column-page-right quarto-layout-panel anchored">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="flourish-embed flourish-chart quarto-layout-cell" data-src="visualisation/7988197?900894" style="flex-basis: 100.0%;justify-content: center;">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
</div>
<div class="panel-caption table-caption">
<p><span class="figure-caption">Table 1. Sequencing capacity varies greatly across US states based on what % of cases are sequenced, how many sequences are shared, and submission latency to GISAID over the past 180 days.</span></p>
</div>
</div>
</section>
<section id="genomic-surveillance-can-help-map-the-pandemic-and-detect-changes." class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="genomic-surveillance-can-help-map-the-pandemic-and-detect-changes.">Genomic surveillance can help map the pandemic and detect changes.</h3>
<p>Genomic surveillance and other epidemiological studies, such as analysis of <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7792982/">air</a> or <a href="https://iwa-network.org/learn/detecting-covid-19-variants-in-wastewater/">wastewater samples,</a> can help determine how prevalent particular variants are, locally and globally, and can help track their circulation through populations. Large-scale surveillance is also essential for detecting new variants with different, perhaps more concerning characteristics. Already, scientists have documented <a href="https://web.musc.edu/about/news-center/2021/09/16/one-hundred-percent-of-latest-covid-samples-sequenced-at-musc-involve-delta-variant">multiple sublineages</a>of the Delta variant, that is, Delta with additional mutations, and one particular sublineage has been designated a <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1029715/technical-briefing-27.pdf">Variant Under Investigation by the UK</a> after it showed signs of slowly outcompeting its parent lineage.</p>
<p>In unvaccinated populations, rampant viral replication and transmission increases the risk of new variants emerging. Among the vaccinated, patterns of <a href="https://www.rockefellerfoundation.org/blog/the-critical-challenge-of-tracking-breakthrough-infections-and-why-we-still-need-better-data-standards/">breakthrough cases</a> can provide information about potentially more transmissible or immune-evasive variants. <strong>It is thus important to sequence a representative sample of virus obtained from both vaccinated and unvaccinated individuals.</strong></p>
<p><strong>Ideally, for effective surveillance, viral genomes from <a href="https://theconversation.com/genomic-surveillance-what-it-is-and-why-we-need-more-of-it-to-track-coronavirus-variants-and-help-end-the-covid-19-pandemic-157540">5% or more</a> of a representative sample of COVID-19 cases should be sequenced.</strong> Over the course of the pandemic, only <a href="https://www.gisaid.org/index.php?id=208">13 countries</a> have sequenced at that level, increasing slightly to a total of 17 in the most recent 6 months. As of the end of November 2021, the United States ranked 20th in the world (of those sharing at least 5,000 genomes with GISAID) over the course of the pandemic, sequencing around 3.6% of its cases. Over the last 6 months, however, the percentage of cases sequenced and submitted to GISAID by the US has risen to nearly 7%, ranking the US 13th in the world and surpassing the 5% surveillance target. This may be an undercount, since the genomes submitted to GISAID may not represent the entirety of the <a href="https://covid.cdc.gov/covid-data-tracker/#published-sars-cov-2-sequences">US sequencing effort</a>. The US Centers for Disease Control and Prevention (CDC) has been <a href="https://covid.cdc.gov/covid-data-tracker/#published-sars-cov-2-sequences">collaborating</a>with <a href="https://www.aphl.org/programs/preparedness/Crisis-Management/COVID-19-Response/Pages/Sequence-Based-Surveillance-Submission.aspx">state and local public health departments</a> as well as commercial diagnostic laboratories to increase sequencing capacity in the US. On November 27, 2021, Scott Becker, the CEO of the Association of Public Health Laboratories, estimated that nationally <a href="https://twitter.com/scottjbecker/status/1464759826345463813">5-10% of all diagnostic specimens</a> were being sequenced, meaning the system can detect variants <a href="https://twitter.com/scottjbecker/status/1464759827381428227">down to 0.01% of circulating viruses</a>. On a <a href="https://www.gisaid.org/index.php?id=209">state-by-state level</a>, however, the percentage of cases sequenced and shared can vary widely, from 1.1% (Puerto Rico) to 34.8% (Vermont) in the last 180 days.</p>
<div id="fig-1" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<p><img src="2nd-Piece-Image-1440x793.png" class="img-fluid figure-img" alt="3 bar charts showing US states by sequencing capacity: genomes shared, % of cases sequenced and shared, median days to deposition of data."></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Sequencing capacity varies greatly across U.S. states based on what % of cases are sequenced, how many sequences are shared, and submission latency to GISAID over the past 180 days.</figcaption><p></p>
</figure>
</div>
<p>A lack of health services often correlates with housing and food insecurity and other factors that can increase an individual’s risk of SARS CoV-2 exposure and thus community transmission of COVID-19. Communities with limited health care infrastructure may also lack testing and vaccination capacity and the ability to sequence viral samples. High levels of viral transmission or the emergence of variants may or may not occur in these communities, but there is no way to know without adequate genomic surveillance. It is crucial to pair robust testing with variant tracking in all communities, to best protect these areas at risk of high spread. <strong>If local outbreaks are identified, interventions such as masking, social distancing, and vaccination can be targeted and decrease the risk of new variants emerging.</strong></p>
</section>
<section id="recent-advances-in-technology-have-made-sequencing-quicker-and-more-affordable." class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="recent-advances-in-technology-have-made-sequencing-quicker-and-more-affordable.">Recent advances in technology have made sequencing quicker and more affordable.</h3>
<p>The United Kingdom began using a new <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243185">PCR genotyping assay</a> in March 2021. This assay uses the same equipment used in diagnostic PCR laboratories while also detecting specific mutations in known variants. The genotyping assay has <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/993879/Variants_of_Concern_VOC_Technical_Briefing_15.pdf">successfully identified variants</a> circulating in the UK. Although it is inexpensive, at <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243185">less than $2 per sample,</a> and has a quicker turnaround time than methods that sequence the whole viral genome, the genotyping assay relies on detecting mutations revealed by prior sequencing of whole genomes and cannot detect new variants that do not possess known mutations. This makes it excellent for following trends of known variants in near real time, but inadequate for comprehensive surveillance.</p>
<p>The cost for sequencing complete genomes has <a href="https://www.nytimes.com/interactive/2021/03/25/magazine/genome-sequencing-covid-variants.html">rapidly declined</a> in the past decade, to as little as <a href="https://www.nature.com/articles/s41467-021-24078-9">$15</a> per sample. The physical size of the equipment has decreased to the point where some sequencing technology is <a href="https://www.pnas.org/content/116/9/3351">portable and processing requirements have fallen to the point that data can be processed on a</a>laptop computer. <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/illumina-dye-sequencing">Second</a>-and <a href="https://nanoporetech.com/applications/dna-nanopore-sequencing">third</a>-generation sequencing technologies put <a href="https://www.mdpi.com/2073-4425/11/8/949">near real-time</a>, <a href="https://www.nature.com/articles/s41587-021-01108-x">on-site sequencing</a> within reach of more laboratories.</p>
<div id="fig-2" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<p><img src="2nd-Piece-Image2-1440x793.png" class="img-fluid figure-img" alt="Scatter plots by US region showing each county as a dot positioned by % popultion fully vaccinated on the x-axis and either % of population in poverty or social vulnerability score on the y-axis. Color of dot indicates large metro, small metro, or micropolitan/rural and radius represents population size."></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: Poorer, more vulnerable, and more rural U.S. counties lag behind in vaccination, including Lincoln Parish, L.A. with 30% in poverty, a medium vulnerability score, and 39% of people fully vaccinated.</figcaption><p></p>
</figure>
</div>
</section>
<section id="obtaining-viral-sequences-from-underrepresented-populations-requires-more-than-just-technology" class="level3">
<h3 class="anchored" data-anchor-id="obtaining-viral-sequences-from-underrepresented-populations-requires-more-than-just-technology">Obtaining viral sequences from underrepresented populations requires more than just technology,</h3>
<p>as illustrated by a project in hard-hit Louisiana. When Covid-19 cases began a Delta-fueled surge in the summer of 2021, <a href="https://www.politico.com/news/2021/09/02/covid-data-gaps-delta-variant-508530">state health officials struggled</a>to get the data they needed. The situation is being improved by a cooperative effort between <a href="https://www.latech.edu/2021/06/14/tech-to-partner-with-gsu-lsuhs-to-track-covid-19-variants/">Louisiana Tech University</a> and <a href="https://www.gram.edu/news/index.php/2021/06/09/the-rockefeller-foundation-announces-key-grants-collaborations-to-strengthen-global-genomic-sequencing-data-sharing/">Grambling State University</a> in Lincoln Parish, along with the Louisiana State University Health Sciences Center Shreveport (LSUHS) in Caddo Parish. The project is one of four <a href="https://www.rockefellerfoundation.org/news/the-rockefeller-foundation-announces-key-grants-and-collaborations-toward-the-creation-of-a-pandemic-prevention-institute/">U.S. Regional Accelerators for Genomic Surveillance</a> supported by the Rockefeller Foundation. The aim of the network, which also includes institutions in Massachusetts, <a href="https://news.wisc.edu/uw-madison-joins-rockefeller-foundations-pandemic-prevention-institute/">Wisconsin</a>, and <a href="https://ufhealth.org/news/2021/uf-health-researchers-join-global-effort-track-covid-19-pandemic-and-others-future">Florida</a>, is to provide <a href="https://www.rockefellerfoundation.org/wp-content/uploads/2021/03/The-Rockefeller-Foundation_Accelerating-National-Genomic-Surveillance.pdf">more complete and equitable regional genomic surveillance</a>.</p>
<p>“What really attracted me to the whole project was the focus on equity,” says Dr.&nbsp;Paul Kim, an Assistant Professor at Grambling, one of 107 historically black colleges and universities in the US. “My role is to try to connect people together so that we can collect specimens that otherwise wouldn’t get sequenced … We’re trying to reach out to the community through people that they know and trust.”</p>
</section>
<section id="genomic-surveillance-is-made-possible-through-collaboration-outreach-and-trust." class="level3">
<h3 class="anchored" data-anchor-id="genomic-surveillance-is-made-possible-through-collaboration-outreach-and-trust.">Genomic surveillance is made possible through collaboration, outreach, and trust.</h3>
<p><a href="https://www.thehealthhut.org/">The Health Hut</a>, a community clinic in Lincoln Parish, Louisiana offers healthcare for its clientele, 65% of whom lack health insurance. The clinic performs a variety of services, including Covid-19 testing and vaccination, and now conducts genomic surveillance. Chelsea Streets, a medical assistant at The Health Hut, helps with obtaining patient swabs for sequencing. “All our patients are very happy to be involved in collecting their samples,” she says. “Anything to help us try to get rid of this Covid.” The process was streamlined by only requiring one swab for both testing and sequencing, eliminating the need for a return visit and making comfort and convenience a priority.</p>
</section>
<section id="engaging-the-community-also-means-empowering-their-clientele-with-information." class="level3">
<h3 class="anchored" data-anchor-id="engaging-the-community-also-means-empowering-their-clientele-with-information.">Engaging the community also means empowering their clientele with information.</h3>
<p>“I love that people are learning about viruses,” says Dr.&nbsp;Jacquelyn White, Medical Director of The Health Hut. “Everybody knows what an antigen is and what an antibody is.” To increase patient access to healthcare, the clinic has Spanish interpreters, as well as counselors to provide mental health support.</p>
<p>Dr.&nbsp;Kim’s laboratory has recently begun performing the sequencing on-site, rather than sending the samples to LSUHS, and has involved students in the research. Colleagues in Grambling’s School of Social Work have been performing outreach to other community clinics and homeless shelters, further promoting community engagement in genomic surveillance. The project has expanded <a href="https://www.latech.edu/2021/11/19/local-collaboration-helps-researchers-everywhere-better-understand-covid-19/">beyond Lincoln Parish</a> to Webster Parish, and even into Mississippi. <a href="https://www.nla-health.com/">A website</a> and <a href="https://www.nla-health.com/dashboard.html">data dashboard</a> keep the community informed of sequencing results. <strong>The success of the program is evident in the dramatic improvement in genomic surveillance in Lincoln Parish, with the percent of samples sequenced <a href="https://www.gram.edu/news/index.php/2021/10/13/gsu-professor-students-involved-in-research-to-study-monitor-coronavirus-in-southern-u-s/">increasing from less than 0.5% to more than 5%</a></strong>.</p>
<p>Community-based programs not only increase vaccine equity but can improve representation of underserved communities in surveillance data and provide a better understanding of the viral landscape to inform public health officials about the state of the pandemic. It also lays the foundation for success with future public health programs.</p>
<p>Knowledge of the prevalence and characteristics of variants in the environment—especially those that can overcome vaccine-induced immunity—will help public health officials and scientists determine the effectiveness of current vaccines, and help develop new vaccines, if necessary. <strong>The need for surveillance will not end with this pandemic, or with coronaviruses. </strong>More comprehensive and efficient methods of monitoring other emerging pathogens can help control future outbreaks before they can become widespread. Building upon local infrastructure and generating community engagement will lay the groundwork so communities are not left behind.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>