<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jacqueline Houtman">
<meta name="author" content="Lindsey Shultz">
<meta name="author" content="Rebecca Glassman">
<meta name="author" content="Jessica Malaty Rivera">
<meta name="author" content="Rick Bright">
<meta name="dcterms.date" content="2021-07-30">

<title>PTC - Making Sense of Variants: They All Deserve Attention. A Few Merit Concern.</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

<script type="text/javascript">

(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-234526530-1', { 'storage': 'none' });

ga('send', {
  hitType: 'pageview',
  'anonymizeIp': true,
});
</script>


</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../ptc-bars.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">PTC</span>
    </a>
  </div>
          <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html">
 <span class="menu-text">Anthology</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../scorecards.html">
 <span class="menu-text">Scorecards</span></a>
  </li>  
</ul>
          <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="../../about.html"><i class="bi bi-info-circle-fill" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/pandemictrack"><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents</h2>
   
  <ul>
  <li><a href="#whats-a-variant" id="toc-whats-a-variant" class="nav-link active" data-scroll-target="#whats-a-variant">What’s a variant?</a></li>
  <li><a href="#how-do-variants-get-foundand-are-they-all-alike" id="toc-how-do-variants-get-foundand-are-they-all-alike" class="nav-link" data-scroll-target="#how-do-variants-get-foundand-are-they-all-alike">How do variants get found–and are they all alike?</a></li>
  <li><a href="#what-happens-when-a-variant-is-foundand-whats-in-a-name" id="toc-what-happens-when-a-variant-is-foundand-whats-in-a-name" class="nav-link" data-scroll-target="#what-happens-when-a-variant-is-foundand-whats-in-a-name">What happens when a variant is found–and what’s in a name?</a></li>
  <li><a href="#whats-a-dominant-lineageand-why-does-it-matter" id="toc-whats-a-dominant-lineageand-why-does-it-matter" class="nav-link" data-scroll-target="#whats-a-dominant-lineageand-why-does-it-matter">What’s a dominant lineage–and why does it matter?</a></li>
  <li><a href="#the-gisaid-initiative" id="toc-the-gisaid-initiative" class="nav-link" data-scroll-target="#the-gisaid-initiative">The GISAID initiative</a></li>
  <li><a href="#what-does-the-delta-variant-say-about-sequencing-and-response-strategy-as-of-mid-2021" id="toc-what-does-the-delta-variant-say-about-sequencing-and-response-strategy-as-of-mid-2021" class="nav-link" data-scroll-target="#what-does-the-delta-variant-say-about-sequencing-and-response-strategy-as-of-mid-2021">What does the Delta variant say about sequencing and response strategy as of mid-2021?</a>
  <ul>
  <li><a href="#india-without-vaccines-or-a-resilient-health-system-surveillance-falters." id="toc-india-without-vaccines-or-a-resilient-health-system-surveillance-falters." class="nav-link" data-scroll-target="#india-without-vaccines-or-a-resilient-health-system-surveillance-falters.">India: Without vaccines or a resilient health system, surveillance falters.</a></li>
  <li><a href="#the-uk-vaccine-coverage-time-to-plan-and-a-coordinated-response." id="toc-the-uk-vaccine-coverage-time-to-plan-and-a-coordinated-response." class="nav-link" data-scroll-target="#the-uk-vaccine-coverage-time-to-plan-and-a-coordinated-response.">The UK: Vaccine coverage, time to plan, and a coordinated response.</a></li>
  <li><a href="#the-us-national-gaps-in-vaccine-coverage-and-sequencing." id="toc-the-us-national-gaps-in-vaccine-coverage-and-sequencing." class="nav-link" data-scroll-target="#the-us-national-gaps-in-vaccine-coverage-and-sequencing.">The US: National gaps in vaccine coverage and sequencing.</a></li>
  </ul></li>
  <li><a href="#how-do-scientists-assess-the-danger-posed-by-any-given-variant" id="toc-how-do-scientists-assess-the-danger-posed-by-any-given-variant" class="nav-link" data-scroll-target="#how-do-scientists-assess-the-danger-posed-by-any-given-variant">How do scientists assess the danger posed by any given variant?</a></li>
  <li><a href="#how-do-vaccines-factor-in" id="toc-how-do-vaccines-factor-in" class="nav-link" data-scroll-target="#how-do-vaccines-factor-in">How do vaccines factor in?</a></li>
  <li><a href="#how-do-surveillance-and-vaccination-work-together-to-curb-the-pandemic" id="toc-how-do-surveillance-and-vaccination-work-together-to-curb-the-pandemic" class="nav-link" data-scroll-target="#how-do-surveillance-and-vaccination-work-together-to-curb-the-pandemic">How do surveillance and vaccination work together to curb the pandemic?</a></li>
  </ul>
</nav>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Making Sense of Variants: They All Deserve Attention. A Few Merit Concern.</h1>
<p class="subtitle lead">Careful surveillance and study of SARS-CoV-2 variants will help determine the public health threats they pose and the mitigation efforts they require.</p>
  <div class="quarto-categories">
    <div class="quarto-category">Variants</div>
  </div>
  </div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Jacqueline Houtman </p>
             <p>Lindsey Shultz </p>
             <p>Rebecca Glassman </p>
             <p>Jessica Malaty Rivera </p>
             <p>Rick Bright </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">July 30, 2021</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>This piece is originally published at <a href="https://www.rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/">rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/</a></p>
</div>
</div>
<ul>
<li>SARS-CoV-2 has already mutated since it was first identified and will continue to do so.</li>
<li>Robust global genomic sequencing is the best tool for tracking existing variants and detecting new ones.</li>
<li>This information enables public health officials, governments, and communities to take action that can save lives. The more high-quality data that is available, the more effective the response.</li>
<li>Even as new variants emerge, vaccines, masking, and social distancing remain critical to ending the Covid-19 pandemic.</li>
</ul>
<p>Every week, it seems, coverage of existing and emerging variants multiplies, with each headline scarier than the last. Terms like “Superstrains,” “Double Mutants,” “Delta Plus” and “Covid on Steroids” are attention-grabbing but do little to convey what countries and communities should expect–or do–in terms of these variant viruses. This document is designed as a guide to what matters, what to watch for, and why.</p>
<p><strong>Bottom line:</strong> Effective surveillance that detects emerging variants is critical to the Covid-19 response. It’s not yet universal, even in countries like the U.S. where there is widespread vaccine access. It will be crucial to scale surveillance along with vaccine access worldwide, since careful study of their characteristics can determine which variants need to be more closely monitored as possible public health threats.</p>
<section id="whats-a-variant" class="level3">
<h3 class="anchored" data-anchor-id="whats-a-variant">What’s a variant?</h3>
<p>All viruses mutate, including SARS-CoV-2, the virus that causes Covid-19. Mutations are the occasional mistakes that occur when a virus replicates–or makes copies of itself–inside host cells. Usually, those mistakes make no difference in how the virus functions. Sometimes the mistakes even weaken the virus, and viruses with those mutations die out. Once in a while, however, an accumulation of advantageous mutations can make a virus better at infecting cells, causing disease, evading the immune system, or spreading within a population, thereby being designated a “variant” of the wild-type (original) virus.</p>
</section>
<section id="how-do-variants-get-foundand-are-they-all-alike" class="level3">
<h3 class="anchored" data-anchor-id="how-do-variants-get-foundand-are-they-all-alike">How do variants get found–and are they all alike?</h3>
<p>Scientists track these SARS-CoV-2 variants, those with mutations that differ in such a way to make them clinically or epidemiologically important. Detecting variants usually requires sequencing the entire genome of a virus, that is, all of its genes. This process takes longer than the diagnostic PCR test, which is designed to detect only small parts of the genome. <strong>Variant surveillance, accomplished by sequencing a representative sample of viral specimens in a population, is essential to determine if new variants are emerging or if known variants are becoming more widespread</strong>. Additional epidemiological, clinical, and laboratory studies are used to assess the properties of variants. Possible <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html">characteristics of variants</a> might include:</p>
<ul>
<li>Interference with diagnostic tests (The <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959426/Variant_of_Concern_VOC_202012_01_Technical_Briefing_5.pdf">Alpha variant</a>, for example, has a genetic change that results in false negative PCR results in one diagnostic test kit.)</li>
<li>Increased ability to bind to or multiply in cells in the laboratory</li>
<li>Increased ability to grow in the laboratory in the presence of antiviral antibodies</li>
<li>Increased community transmission within a population</li>
<li>Reduced effectiveness of therapeutic drugs such as monoclonal antibodies</li>
<li>Increased incidence or severity of disease in infected individuals</li>
<li>Ability to infect previously infected individuals (reinfection)</li>
<li>Ability to infect fully vaccinated individuals (<a href="https://www.rockefellerfoundation.org/blog/the-u-s-can-lead-the-way-in-vaccine-breakthrough-reporting-will-it-squander-this-opportunity/">vaccine breakthrough</a>)</li>
</ul>
</section>
<section id="what-happens-when-a-variant-is-foundand-whats-in-a-name" class="level3">
<h3 class="anchored" data-anchor-id="what-happens-when-a-variant-is-foundand-whats-in-a-name">What happens when a variant is found–and what’s in a name?</h3>
<p>The World Health Organization (WHO) is monitoring more than <a href="https://www.cnbc.com/2021/06/16/who-says-delta-covid-variant-has-now-spread-to-80-countries-and-it-keeps-mutating.html">50 different variants of the SARS-CoV-2 virus</a>. Most are unlikely to be problematic, but a handful can be classified by the U.S. Centers for Disease Control and Prevention (<a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-classifications.html?CDC_AA_refVal=https%3A%2F%2Fwww.cdc.gov%2Fcoronavirus%2F2019-ncov%2Fvariants%2Fvariant-info.html">CDC</a>) and <a href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">WHO</a> into four categories, based on genetic or epidemiological evidence:</p>
<ul>
<li><strong>Alerts for Further Monitoring</strong> are variants with genetic markers that are suspected to affect virus characteristics but require enhanced monitoring and evidence to assess their epidemiological impact. Former variants of concern and variants of interest may be monitored for an extended period of time under this classification. Some organizations designate variants in this category as <a href="https://www.ecdc.europa.eu/en/covid-19/variants-concern">Variants Under Monitoring (VUM)</a>.</li>
<li><strong>Variants of Interest (VOI)</strong> are those with genetic markers predicted or known to affect virus characteristics and identified in multiple cases of Covid-19, in clusters or in multiple countries.</li>
<li><strong>Variants of Concern (VOC)</strong> are variants that have been demonstrated to be more transmissible, cause more severe disease, or make vaccines, diagnostic tests, medications, or public health measures less effective.</li>
<li><strong>Variants of High Consequence (VOHC)</strong> are those that cause significant reductions in the effectiveness of diagnostic tests, vaccines, or therapeutic drugs or cause significantly more severe disease. As of mid-July 2021, no variant has been designated a VOHC.</li>
</ul>
<p>Groups of scientists name variants with different strings of letters and numbers that trace their genetic lineages, which can be confusing to the public. News organizations and others addressing non-scientific audiences have attempted to simplify the naming of variants by referring to the countries in which they first emerged, but that can result in stigma, blame, or prejudice. In an effort to identify viruses in easier-to-pronounce and non-stigmatizing ways, the WHO recently proposed a new system of labeling variants of interest and concern using letters of the Greek alphabet.</p>
<div id="tbl-1" class="tbl-parent quarto-layout-panel anchored">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<table class="table">
<colgroup>
<col style="width: 10%">
<col style="width: 26%">
<col style="width: 20%">
<col style="width: 16%">
<col style="width: 26%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">WHO label</th>
<th style="text-align: left;">Pango Lineage</th>
<th style="text-align: left;">GISAID clade/lineage</th>
<th style="text-align: left;">Nextstrain clade</th>
<th style="text-align: left;">Earliest documented samples</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Alpha</td>
<td style="text-align: left;">B.1.1.7</td>
<td style="text-align: left;">GRY</td>
<td style="text-align: left;">20I (V1)</td>
<td style="text-align: left;">United Kingdom, Sep-2020</td>
</tr>
<tr class="even">
<td style="text-align: left;">Beta</td>
<td style="text-align: left;">B.1.351 B.1.351.2 B.1.351.3</td>
<td style="text-align: left;">GH/501Y.V2</td>
<td style="text-align: left;">20H (V2)</td>
<td style="text-align: left;">South Africa, May-2020</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Gamma</td>
<td style="text-align: left;">P.1 P.1.1 P.1.2</td>
<td style="text-align: left;">GR/501Y.V3</td>
<td style="text-align: left;">20J (V3)</td>
<td style="text-align: left;">Brazil, Nov-2020</td>
</tr>
<tr class="even">
<td style="text-align: left;">Delta</td>
<td style="text-align: left;">B.1.617.2 AY.1 AY.2</td>
<td style="text-align: left;">G/478K.V1</td>
<td style="text-align: left;">21A</td>
<td style="text-align: left;">India, Oct-2020</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="panel-caption table-caption">
<p><span class="figure-caption">Table 1. Current Variants of Concern (VOC), as designated by the WHO</span></p>
</div>
</div>
<p>Each country can classify variants based on local prevalence and transmission rates, and the WHO can determine if variants are of global interest or concern. Variant designations can change, as well. For example, the <a href="https://covariants.org/variants/S.L452R">Epsilon variant</a> was designated a VOC by the CDC, but not by the WHO, because it was not widespread outside of the U.S. and Mexico. The CDC has since downgraded Epsilon to a VOI. On <a href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---6-july-2021">July 6, 2021</a>, the WHO recognized four VOIs (Eta, Iota, Kappa, and Lambda), and reclassified three former VOIs (Epsilon, Zeta, and Theta) as “Alerts for further monitoring.” Variant classifications may escalate or de-escalate based on emerging evidence, including a change in relative risk compared to other circulating variants.</p>
<p>These designations reflect evidence-informed predictions of the risks a variant might pose to individual and public health, which can be obscured by media coverage. It is nearly impossible to predict exactly how a new mutation or a new variant will affect the spread of Covid-19. Careful study of the data, however, can help public health authorities track, or even control variants that emerge. Global surveillance of emerging variants can enable vaccine developers to monitor the likelihood of vaccine breakthrough infections and consider strategies for booster shots and new vaccines in the future.</p>
</section>
<section id="whats-a-dominant-lineageand-why-does-it-matter" class="level3">
<h3 class="anchored" data-anchor-id="whats-a-dominant-lineageand-why-does-it-matter">What’s a dominant lineage–and why does it matter?</h3>
<p>In the United States, at least 33 states report variant data publicly on a regular basis. The presence of variants is usually expressed as a proportion of total samples sequenced. The <a href="https://covid.cdc.gov/covid-data-tracker/#variant-proportions">CDC estimates</a> that as of July 3, 28.7% of viruses sequenced in the US were variants of concern of the Alpha lineage, 51.7% were of the Delta lineage and 8.9% were of the Gamma lineage. The distribution of variants across the US is not uniform, and <a href="https://covid.cdc.gov/covid-data-tracker/#variant-proportions">significant regional differences</a> exist.</p>
<p>All VOI and VOC worldwide merit close observation. Some remain restricted to outbreaks in limited geographical areas. Others can rapidly spread worldwide, displacing other circulating viruses and becoming what’s known as the “dominant lineage.” The Alpha variant, for example, was first detected in the UK in September 2020 and became the dominant lineage there by December 2020. By spring, the Alpha variant was detected in <a href="https://ophrp.org/upload/pdf/j-phrp-2021-0037.pdf">over 150 countries</a> and had become the dominant lineage in the US. The Delta variant is on track to have a similar global effect, playing a significant role in the devastating spring surge in India and displacing the Alpha variant as the dominant lineage in the UK in May 2021 and US in July 2021. The Delta variant of SARS-CoV-2 was upgraded from a VOI to a VOC <a href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">by the WHO on May 11,</a> and <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html#Concern">by the CDC on June 15</a>, 2021. Its spread during early 2021 can be examined from India, where it was first discovered, through the UK where it was introduced relatively early via travel, and to the US where it arose as the dominant global variant. A robust approach to sequencing variants and tracking their spread allows countries to understand and anticipate the arrival of dominant variants, which may be more transmissible, as is the case with the Delta variant, or have different impacts on disease progression. Sequencing allows countries to remain one step ahead of variants, which can be crucial for planning and response.</p>
</section>
<section id="the-gisaid-initiative" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="the-gisaid-initiative">The GISAID initiative</h3>
<p>The <a href="https://www.gisaid.org/">GISAID</a> Initiative promotes the rapid sharing of data from all influenza viruses and the coronavirus causing Covid-19. This includes genetic sequence and related clinical and epidemiological data associated with human viruses, and geographical as well as species-specific data associated with avian and other animal viruses, to help researchers understand how viruses evolve and spread during epidemics and pandemics. GISAID’s EpiCoV™ database provides by far the largest global collection of genetic sequences and associated metadata of SARS-CoV-2, with about 2.25 million entries as of July 2021. The Rockefeller Foundation is working to expand global genomic sequencing capacity, with plans to collaborate with GISAID, advancing its pathogen data-sharing platform to enable more rapid sharing of data to trigger public health action and development of vaccines and diagnostic tests.</p>
<div id="fig-1" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<div class="flourish-embed flourish-chart" data-src="visualisation/6680291?900894">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Global variant trends</figcaption><p></p>
</figure>
</div>
</section>
<section id="what-does-the-delta-variant-say-about-sequencing-and-response-strategy-as-of-mid-2021" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="what-does-the-delta-variant-say-about-sequencing-and-response-strategy-as-of-mid-2021">What does the Delta variant say about sequencing and response strategy as of mid-2021?</h3>
<section id="india-without-vaccines-or-a-resilient-health-system-surveillance-falters." class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="india-without-vaccines-or-a-resilient-health-system-surveillance-falters.">India: Without vaccines or a resilient health system, surveillance falters.</h4>
<p>The progenitor of the Delta variant (B.1.617) was first identified in late 2020 in India. Of the three variants derived from that progenitor, one (Kappa, or B.1.617.1) has been designated a VOI by the WHO and another (Delta, or B.1.617.2) is designated a VOC. Between January and April 2021, the Indian SARS-CoV-2 sequencing consortium (INSACOG) was sequencing approximately <a href="https://www.nature.com/articles/d41586-021-01427-8">0.75%</a> of positive cases (although the <a href="https://www.nature.com/articles/d41586-021-01427-8">original stated target was to randomly sequence 5%</a>). In March 2021, INSACOG identified Delta as a public health threat, due to a spike in cases and the presence of several concerning <a href="https://www.nature.com/articles/d41586-021-01427-8">mutations</a> in samples taken in the prior three months. The massive spring increase in cases in India, almost all of which were the <a href="https://www.nature.com/articles/d41586-021-01274-7">due in large part to the Delta variant, which spread rapidly in the context of</a> and low vaccination rates, overwhelmed already strained healthcare systems throughout the country. This burden greatly reduced the percentage of samples sequenced. It’s critical to pair investments in health systems and surveillance, so that epidemic surges trigger vaccination, care and treatment without letting sequencing lag behind.</p>
<p>Based on the limited available data, by <a href="https://www.ecdc.europa.eu/sites/default/files/documents/Emergence-of-SARS-CoV-2-B.1.617-variants-in-India-and-situation-in-the-EUEEA_0.pdf">mid-April 2021</a>, Delta became the dominant lineage of SARS-CoV-2 in India. Assuming the most recent two weeks of data is incomplete, data derived from <a href="https://www.gisaid.org">GISAID</a> on 6/28 indicates India sequenced an average of 0.04% of its Covid-19 cases in the month of June. During this time, their median number of days (median lag time) from specimen collection to sequencing and GISAID submission was 17 days. There are many variables affecting turnaround time. The vast majority of sites worldwide lack sequencing capabilities, for example, and it takes time to transfer the samples to sequencing laboratories. Nevertheless, a target of less than seven days is reasonable and achievable in many places. <strong>A functioning healthcare infrastructure, adequate and equitable vaccine supplies, and higher sequencing capacity are essential for rapid and effective pandemic response.</strong></p>
<div id="fig-2" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<div class="flourish-embed flourish-chart" data-src="visualisation/6680680?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-scatter" data-src="visualisation/6657806?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-chart" data-src="visualisation/6693629?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: India variant trends</figcaption><p></p>
</figure>
</div>
</section>
<section id="the-uk-vaccine-coverage-time-to-plan-and-a-coordinated-response." class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="the-uk-vaccine-coverage-time-to-plan-and-a-coordinated-response.">The UK: Vaccine coverage, time to plan, and a coordinated response.</h4>
<p>The Delta variant was first detected in the UK in February 2021. Between March 25 and April 7, 5.1% of travellers from India, 3.7% of those from Bangladesh, and 6.2% of those from Pakistan tested positive for Covid-19. In response, the government placed <a href="https://www.bbc.com/news/56801288">strict restrictions</a> on travel from <a href="https://www.gov.uk/government/news/countries-added-to-red-list-to-protect-uk-against-variants-of-concern">Pakistan and Bangladesh</a> on April 9 and from <a href="https://www.gov.uk/government/news/india-added-to-red-list-as-cases-of-new-variant-rise">India</a> on April 23. Of those travellers who had been in these countries in the previous ten days, only UK citizens or legal residents were allowed entry, and they were subject to a mandatory ten-day quarantine and repeated testing for SARS-CoV-2. These policies presented the UK with additional time to gather data and prepare a response to the new variant. The backbone of an effective CovidOVID-19 response is widespread access to and uptake of effective vaccines. This is a top priority for resources and national programming. To protect the gains made in countries, such as the UK, where vaccination rates are high, near universal screening and sequencing of travelers with rigorous quarantine measures and contact tracing can help protect progress made. This suite of approaches will be critical to employ as variants emerge and spread worldwide.</p>
<p>The UK experience offers a positive example. The country has a robust sequencing program and recently introduced a new, cost-effective rapid <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243185">genotyping</a> <a href="https://www.gov.uk/government/news/groundbreaking-new-technology-to-detect-known-variants-of-concern">assay</a>, which has greatly <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/994839/Variants_of_Concern_VOC_Technical_Briefing_16.pdf">increased the volume and speed</a> of identification of variants, with a turnaround time of <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/993879/Variants_of_Concern_VOC_Technical_Briefing_15.pdf">12-24 hours</a>, compared to <a href="https://www.gisaid.org/index.php?id=208">5-23 days</a> for traditional sequencing techniques. Assuming the most recent two weeks of data is incomplete, data derived from <a href="https://www.gisaid.org">GISAID</a> indicates the UK has sequenced an average of 41% of its Covid-19 cases in the prior four weeks, with a median lag time of 10 days. Delta overcame Alpha as the predominant variant in May 2021, and a Delta-driven uptick in new cases has prompted a <a href="https://www.reuters.com/world/uk/uks-johnson-set-announce-delay-end-covid-restrictions-2021-06-13/">delay</a> in lifting remaining Covid-19 social restrictions from June 21 to July 19, 2021. That time will be used to ramp up vaccinations. The Delta variant now accounts for <a href="https://www.gov.uk/government/news/confirmed-cases-of-covid-19-variants-identified-in-uk">99%</a> of sequenced cases and <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1001009/Variants_of_Concern_VOC_Technical_Briefing_18.pdf">97%</a> of genotyped cases in the UK. The comprehensive nature of its sequencing program has allowed the UK to respond in a timely and tailored manner to evolving data on the ground.</p>
<div id="fig-3" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<div class="flourish-embed flourish-chart" data-src="visualisation/6696914?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-scatter" data-src="visualisation/6696927?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-chart" data-src="visualisation/6696961?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;3: UK variant trends</figcaption><p></p>
</figure>
</div>
</section>
<section id="the-us-national-gaps-in-vaccine-coverage-and-sequencing." class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="the-us-national-gaps-in-vaccine-coverage-and-sequencing.">The US: National gaps in vaccine coverage and sequencing.</h4>
<p>Sequencing has been less robust in the US. The Delta variant was first detected in the US in <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant.html">March 2021</a>. Although its prevalence as of July 7, 2021 is lower in the US than the UK, its numbers are rising. <a href="https://covid.cdc.gov/covid-data-tracker/#variant-proportions">Genomic surveillance in the US</a> indicated that the Delta variant accounted for less than 1% of cases as of April 4, doubling approximately every two weeks and accounting for more than 51% of cases as of July 3. Just as the Alpha variant became the dominant lineage in the US in <a href="https://www.nytimes.com/2021/02/07/health/coronavirus-variant-us-spread.html">mid-March</a>, in large part because it was more transmissible than the prevalent lineages at the time, the even more transmissible Delta variant is replacing Alpha. As scientists predicted, Delta <a href="https://www.reuters.com/world/us/delta-variant-already-dominant-us-cdc-estimates-show-2021-07-07/">has become the dominant lineage</a>in the US. Parts of <a href="https://www.kcmo.gov/Home/Components/News/News/1636/625">Missouri</a>, <a href="https://www.nytimes.com/live/2021/07/07/world/covid-19-vaccine-coronavirus-updates-#nevada-covid-delta">Nevada</a>, <a href="https://www.latimes.com/california/story/2021-07-08/coronavirus-cases-in-california-rise-for-first-time-in-months-as-delta-variant-spreads">California</a>, and <a href="https://www.denverpost.com/2021/07/08/colorado-delta-variant-country-jam/">Colorado</a> are seeing Delta-related surges in Covid-19 cases.</p>
<div id="fig-4" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<div class="flourish-embed flourish-chart" data-src="visualisation/6697064?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-scatter" data-src="visualisation/6697098?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-chart" data-src="visualisation/6697144?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;4: US variant trends</figcaption><p></p>
</figure>
</div>
<p>Throughout 2020, laboratories in the US lacked the resources and infrastructure needed for effective genomic surveillance of SARS-CoV-2 variants. In December 2020, the US ranked <a href="https://www.washingtonpost.com/world/2020/12/23/us-leads-world-coronavirus-cases-ranks-43rd-sequencing-check-variants/">43rd in the world</a>, sequencing only 0.3% of samples. With increased federal <a href="https://jamanetwork.com/journals/jama/fullarticle/2777988">funding</a> and <a href="https://jamanetwork.com/journals/jama/fullarticle/2776739">coordination</a>, beginning in early 2021, the capacity and speed of genomic sequencing have ramped up, along with an increased scrutiny on the scope of data being made available to the public. In the United States as of June 2021, at least 33 states reported variant data publicly on a regular basis. Assuming incomplete data for the two weeks prior, data derived from <a href="https://www.gisaid.org">GISAID</a> on 6/28 indicates the US as a whole sequenced nearly 4.8% of its Covid-19 cases in the month of June, with a median lag time of 15 days. As of <a href="https://www.gisaid.org/index.php?id=209">July 21</a>, states’ median percent sequenced in the past 30 days ranged from 0% (South Dakota) to 13.28% (Nebraska) and the median lag time ranged from 7 days (Kansas and Nebraska) to 23 days (Delaware and Vermont), illustrating the differences in sequencing capacity between states. Although this is a slight improvement, US sequencing efforts are still insufficient to detect and monitor variants that emerge and circulate. Scientists have recommended sequencing between <a href="https://www.medrxiv.org/content/10.1101/2021.01.12.21249613v1.full">5%</a> to <a href="https://magazine.jhsph.edu/2021/covid-19-how-track-viral-variants">30%</a> of cases.</p>
<p>As of July 20, 2021, the Delta variant has been detected in <a href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---20-july-2021">124</a> countries with a prevalence of <a href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---20-july-2021">over 75%</a> in more than a dozen countries. Case numbers are expected to increase, especially in areas with low vaccination rates and insufficient genomic surveillance. With just 1% of people vaccinated, <a href="https://www.washingtonpost.com/world/2021/07/09/coronavirus-latest-updates/">cases in Africa</a> are doubling every 18 days, with Namibia, Uganda, Zambia, and South Africa most severely affected. <a href="https://www.cidrap.umn.edu/news-perspective/2021/07/delta-covid-variant-surges-asia-casts-shadow-olympics">Indonesia, Thailand, and South Korea</a> are seeing record number of cases, and the Delta variant became dominant in <a href="https://covariants.org/per-country">Japan</a> before the Olympic Games Opening Ceremony on July 23, prompting Olympic officials to ban all spectators. Parts of <a href="https://www.npr.org/2021/06/29/1011201162/coronavirus-surges-arnound-the-world-are-linked-to-delta-variant">Australia</a> are locking down due to Delta outbreaks.</p>
</section>
</section>
<section id="how-do-scientists-assess-the-danger-posed-by-any-given-variant" class="level3">
<h3 class="anchored" data-anchor-id="how-do-scientists-assess-the-danger-posed-by-any-given-variant">How do scientists assess the danger posed by any given variant?</h3>
<p>All <a href="https://abcnews.go.com/Health/delta-covid-19-variant-now-80-countries/story?id=78339159">variants</a>, including Delta, are assessed for transmissibility, severity of disease, and ability to evade the immune system. In a <em>Washington Post</em> <a href="https://www.washingtonpost.com/opinions/2021/06/16/delta-variant-ashish-jha-us-vaccines/">editorial</a>, Dr.&nbsp;Ashish K. Jha, Dean of the Brown University School of Public Health called the Delta variant “one of the first triple threats across all those factors.” Studies have shown that the Delta variant is <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/993879/Variants_of_Concern_VOC_Technical_Briefing_15.pdf">40-60% more transmissible</a> than the Alpha variant, which is in turn <a href="https://science.sciencemag.org/content/372/6538/eabg3055">43-90%</a> more transmissible than the original virus. The increased transmissibility of Delta may be related to a <a href="https://www.npr.org/sections/goatsandsoda/2021/07/08/1013794996/why-the-delta-variant-is-so-contagious-a-new-study-sheds-light">1,000-fold increase</a> in the amount of virus in the respiratory tract, compared to the original virus. Infection with the Delta variant is <a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(21)01358-1/fulltext?utm_campaign=lancetcovid21&amp;utm_source=twitter&amp;utm_medium=social">twice as likely</a> as the Alpha variant to result in an ER visit or hospitalization. Reports out of <a href="https://www.nytimes.com/2021/06/12/world/china-covid-delta-variant-guangzhou.html">Guangzhou, China</a> suggest an increase in the severity of disease caused by the Delta variant. A <a href="https://covid.joinzoe.com/post/new-top-5-covid-symptoms">study in the UK</a> found that, as of June 23, 2021, the most common symptoms of Covid-19 in unvaccinated people are headache and runny nose, with shortness of breath and loss of the sense of smell being much rarer, ”indicating the symptoms as recorded previously are changing with the evolving variants of the virus.”</p>
</section>
<section id="how-do-vaccines-factor-in" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="how-do-vaccines-factor-in">How do vaccines factor in?</h3>
<p>Every variant of concern is evaluated to assess how well it is neutralized by antibodies from vaccinated individuals. Clinical studies assess how well vaccines perform against infection or disease caused by variants. So far, the information about vaccine-induced protection against variants has been reassuring for the people and places where vaccines are available. Fully vaccinated individuals mount a robust immune response that provides <a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(21)01358-1/fulltext">good protection</a> against all known variants, including Delta. Those who have only received the first dose of a two-dose regimen, however, have reduced protection against the Delta variant compared to other known variants. Some studies show efficacy against symptomatic disease as low as <a href="https://www.bmj.com/content/373/bmj.n1346">33%</a> in singly vaccinated individuals, however studies also indicate around <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/997414/Variants_of_Concern_VOC_Technical_Briefing_16.pdf">75% vaccine effectiveness</a> is maintained against hospitalization, even after only one dose. (Studies on the single-dose Johnson &amp; Johnson/Janssen vaccine are <a href="https://apps.who.int/iris/bitstream/handle/10665/341784/WHO-2019-nCoV-vaccines-SAGE-recommendation-Ad26.COV2.S-2021.2-eng.pdf?sequence=2">ongoing</a>, and while some <a href="https://www.biorxiv.org/content/10.1101/2021.07.19.452771v1">conflicting findings</a> exist, early <a href="https://www.nejm.org/doi/full/10.1056/NEJMc2108829">data</a> showed promising <a href="https://www.jnj.com/positive-new-data-for-johnson-johnson-single-shot-covid-19-vaccine-on-activity-against-delta-variant-and-long-lasting-durability-of-response">results</a>, especially against severe disease.) It is important to note that many differences exist in these studies, including methodological differences as well as the populations and vaccines involved, and additional clinical and laboratory studies are needed to provide a <a href="https://www.who.int/publications/m/item/weekly-epidemiological-update-on-covid-19---20-july-2021">clearer picture</a>.</p>
<p>In addition to the biological complexities emerging variants can create, improving vaccination rates remain a notable challenge. Worldwide, <a href="https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html">fifteen countries</a> have more than 50% of residents fully vaccinated as of July 20. In the US, only 48.7% of the population is <a href="https://covid.cdc.gov/covid-data-tracker/#vaccinations">fully vaccinated</a> as of July 20, but states range from 33.7% (Alabama) to 67% (Vermont). On a <a href="https://www.nytimes.com/interactive/2020/us/covid-19-vaccine-doses.html">county-by-county</a> basis, the variation in vaccination rates is even greater. This situation can lead to local <a href="https://www.bloomberg.com/news/articles/2021-06-16/pockets-of-unvaccinated-americans-threaten-to-prolong-pandemic">pockets of increased transmission</a>, resulting in more cases, more viral replication, and more opportunities for new mutations and new variants to emerge.</p>
<p><strong>The immense and preventable delay in scaling global access to vaccinations is also the time window in which variants can continue to evolve</strong>. A <a href="https://www.sciencemediacentre.org/expert-reaction-to-cases-of-the-b-1-617-2-delta-variant-with-the-k417n-mutation/">new subtype</a> derived from the Delta variant has already been detected in India, the UK, Portugal, and the US. Dubbed B.1.617.2.1, AY.1, or, more colloquially “Nepal Variant” or “Delta Plus,” it possesses an additional mutation (K417N) that is also present in the Beta variant. This mutation has been associated with an <a href="https://www.sciencedirect.com/science/article/pii/S0888754321001798">increased ability</a> to <a href="https://www.theguardian.com/world/2021/jun/03/nepal-covid-variant-does-it-exist-should-concerned">evade the immune system</a>. That mutation, combined with the increased transmissibility of Delta, has raised the concern of scientists and this virus subtype is being closely watched, although it has not yet been given a designation separate from the Delta VOC. Fortunately, it can be identified with the new <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/994839/Variants_of_Concern_VOC_Technical_Briefing_16.pdf">rapid genotyping assay</a>, which will aid in surveillance.</p>
<div id="fig-5" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<div class="flourish-embed flourish-chart" data-src="visualisation/6697196?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<div class="flourish-embed flourish-chart" data-src="visualisation/6700372?718630">
<script src="https://public.flourish.studio/resources/embed.js"></script>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;5: Percent of new cases sequenced and submission lag per specimen collection week</figcaption><p></p>
</figure>
</div>
</section>
<section id="how-do-surveillance-and-vaccination-work-together-to-curb-the-pandemic" class="level3">
<h3 class="anchored" data-anchor-id="how-do-surveillance-and-vaccination-work-together-to-curb-the-pandemic">How do surveillance and vaccination work together to curb the pandemic?</h3>
<p>Globally, there is a wide range in the percent of positive Covid-19 samples that are sequenced; some countries sequence <a href="https://coronavirus.jhu.edu/data/variant-data">less than 0.01%, while some sequence more than 77%</a>. The reasons for these disparities are many, including resources, technology, infrastructure, and policy decisions. There is no definitive way to predict where the next variant will emerge, and without ample sequencing, variants can become widespread before they are even detected. In addition, the lag time between obtaining a sample, determining its sequence, and reporting that data publicly needs to be shortened. <strong>A lag time of weeks, or even months, enables emerging variants to circulate widely before mitigation measures can be taken.</strong> It is also important to note that the samples chosen to sequence are rarely representative. <a href="https://outbreak.info/situation-reports/caveats">Sampling bias</a> can result from differences in sequencing capacity and lag times, as well as selective sampling during outbreaks and among travellers. Worldwide increases in sequencing capacity and speed are necessary for effective variant surveillance.</p>
<p><strong>The best way to interrupt the transmission cycle is full vaccination, which protects those who are vaccinated, as well as unvaccinated or under-vaccinated members of their <a href="https://www.nejm.org/doi/full/10.1056/NEJMc2107717">households</a> and <a href="https://www.nature.com/articles/s41591-021-01407-5">communities</a>.</strong> Vaccination rates, like sequencing rates, vary widely between (and within) countries. Increasing those rates requires international cooperation. Countries, where variants are detected, should be helped, not stigmatized. A country with high testing, sequencing, and vaccination rates may be at an advantage, but it may only be temporary. If the rest of the world is not included in surveillance and vaccination efforts, there is little chance of ending the current pandemic or preventing the next one.</p>
<p>Attention-grabbing headlines about variants may get more clicks and airtime, but they do little to promote public health and mitigate community risk. Detecting new or more widespread variants should trigger action, not panic. <strong>An outbreak should result in more testing and sequencing, as well as enhanced vaccination efforts and non-pharmaceutical interventions such as masking and social restrictions on a personal and community level.</strong> Global cooperative surveillance and careful study of the properties of viral variants will provide the information we need to react in a fact-based rather than fear-based way to any new “supervirus.”</p>
<hr>
<p>We are grateful to the data contributors who shared the data used in this Web Application via the GISAID Initiative<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a><sup>,</sup><a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a>: the Authors, the Originating Laboratories responsible for obtaining the specimens, and the Submitting Laboratories that generated the genetic sequences and metadata.</p>
<p>GISAID data provided on this website are subject to GISAID <a href="https://www.gisaid.org/DAA">Terms and Conditions</a>.</p>


</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>Elbe, S., and Buckland-Merrett, G. (2017) Data, disease and diplomacy: GISAID’s innovative contribution to global health. Global Challenges, 1:33-46. DOI: <a href="http://dx.doi.org/10.1002/gch2.1018">10.1002/gch2.1018</a> PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6607375/">31565258</a><a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>Shu, Y., McCauley, J. (2017) GISAID: From vision to reality. EuroSurveillance, 22(13). DOI: <a href="http://dx.doi.org/10.2807/1560-7917.ES.2017.22.13.30494">10.2807/1560-7917.ES.2017.22.13.30494</a> PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388101/">PMC5388101</a><a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>