<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jacqueline Houtman">
<meta name="author" content="Lindsey Shultz">
<meta name="author" content="Jessica Malaty Rivera">
<meta name="author" content="Emily Bass">
<meta name="author" content="Rick Bright">
<meta name="author" content="Dave Luo">
<meta name="author" content="Jonathan Gilmour">
<meta name="dcterms.date" content="2021-12-28">

<title>PTC - Vaccine Inequity Increases the Risk of New SARS-CoV-2 Variants Emerging</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

<script type="text/javascript">

(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-234526530-1', { 'storage': 'none' });

ga('send', {
  hitType: 'pageview',
  'anonymizeIp': true,
});
</script>


</head>

<body class="floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../ptc-bars.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">PTC</span>
    </a>
  </div>
          <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html">
 <span class="menu-text">Anthology</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../scorecards.html">
 <span class="menu-text">Scorecards</span></a>
  </li>  
</ul>
          <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="../../about.html"><i class="bi bi-info-circle-fill" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/pandemictrack"><i class="bi bi-twitter" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
            <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Contents</h2>
   
  <ul>
  <li><a href="#vaccination-rates" id="toc-vaccination-rates" class="nav-link active" data-scroll-target="#vaccination-rates">Vaccination rates</a></li>
  <li><a href="#sequencing-capacity" id="toc-sequencing-capacity" class="nav-link" data-scroll-target="#sequencing-capacity">Sequencing capacity</a></li>
  <li><a href="#omicron-sequences" id="toc-omicron-sequences" class="nav-link" data-scroll-target="#omicron-sequences">Omicron sequences</a></li>
  </ul>
</nav>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Vaccine Inequity Increases the Risk of New SARS-CoV-2 Variants Emerging</h1>
<p class="subtitle lead">Robust genomic surveillance, along with transparency, communication, and global collaboration, is needed to detect and control emerging variants.</p>
  <div class="quarto-categories">
    <div class="quarto-category">Vaccine Equity</div>
    <div class="quarto-category">Variants</div>
  </div>
  </div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Jacqueline Houtman </p>
             <p>Lindsey Shultz </p>
             <p>Jessica Malaty Rivera </p>
             <p>Emily Bass </p>
             <p>Rick Bright </p>
             <p>Dave Luo </p>
             <p>Jonathan Gilmour </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">December 28, 2021</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>This piece is originally published at <a href="https://www.rockefellerfoundation.org/case-study/vaccine-inequity-increases-the-risk-of-new-sars-cov-2-variants-emerging/">rockefellerfoundation.org/case-study/vaccine-inequity-increases-the-risk-of-new-sars-cov-2-variants-emerging/</a></p>
</div>
</div>
<p>To control the Covid-19 pandemic, the World Health Organization (WHO) called on member states to meet <a href="https://reliefweb.int/report/world/opening-remarks-dr-tedros-adhanom-ghebreyesus-who-director-general-special-session">a goal</a> of vaccinating 40% of their population by the end of 2021. As of mid December 2021, <a href="https://ourworldindata.org/covid-vaccinations">over 8.6 billion</a>doses have been administered globally. Unfortunately, the distribution of vaccines has been far from uniform, with the percentage of populations fully vaccinated ranging from <a href="https://ourworldindata.org/covid-vaccinations">more than 90% to less than 1%</a>. Predictably, those countries on the high end tend to be wealthy nations, while those at the low end tend to be <a href="https://blogs.worldbank.org/opendata/new-world-bank-country-classifications-income-level-2021-2022">low- and lower-middle income countries</a>. In fact, <a href="https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html">73%</a> of all vaccines administered have gone to high- and upper-middle-income countries. <a href="https://ourworldindata.org/covid-vaccination-global-projections">Fewer than 20% of African countries are on track</a> to meet the 40% goal, and <a href="https://www.rockefellerfoundation.org/blog/genomic-surveillance-is-essential-to-track-covid-19-variants-in-both-unvaccinated-and-vaccinated-populations/">6 times as many booster doses</a>have been administered worldwide as first doses in low income countries. This makes the <a href="https://www.devex.com/news/will-countries-reach-the-70-vaccination-target-by-mid-2022-102156">challenge of reaching</a> the WHO’s next <a href="https://www.who.int/news/item/07-10-2021-who-un-set-out-steps-to-meet-world-covid-vaccination-targets#:~:text=The%20new%20strategy%20outlines%20a,70%25%20by%20mid%2D2022.">milestone of vaccinating 70% of the populations of all countries by mid-2022</a> even more daunting. <strong>The inequitable distribution of vaccines is not just a practical problem for undervaccinated countries. </strong>Beyond being a moral imperative, <a href="https://www.rockefellerfoundation.org/news/joint-statement-foundations-decry-vaccine-inequity-leading-to-the-emergence-and-circulation-of-dangerous-variants/">vaccine equity is a global concern.</a> As long as these inequities persist anywhere in the world, everyone, regardless of where they live or their own vaccination status, will continue in the current state of global crisis.</p>
<section id="vaccination-rates" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="vaccination-rates">Vaccination rates</h4>
<div id="fig-1" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<p><img src="ptc2-viz1-final-1440@2x-2048x1128.png" class="img-fluid figure-img" alt="Bar chart showing share of world population with at least one dose of COVID-19 vaccine colored by country income level."></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: Vaccination rates by population size and country income level. High and upper middle income countries have vaccinated far more of their populations than have most low and lower middle income countries.</figcaption><p></p>
</figure>
</div>
<p>Global inequities in access to Covid-19 vaccines reflect entrenched flaws in current approaches to financing for health, <a href="https://www.ghsindex.org/">pandemic preparedness</a>, and the <a href="https://www.sciencedirect.com/science/article/pii/S0925527321001699">vaccine supply chain</a>. <a href="https://www.doctorswithoutborders.org/what-we-do/news-stories/news/one-year-after-first-shot-moderna-and-pfizer-must-urgently-share-covid">Pharmaceutical companies</a> resist technology transfer and waivers of trade-related intellectual property. <a href="https://www.un.org/press/en/2021/sgsm20986.doc.htm">Vaccine “nationalism”</a> sees high-income countries hoarding doses. <a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(13)62105-4/fulltext">Investment</a> in the human resources and infrastructure needed for resilient health systems has lagged for decades. <a href="https://www.nytimes.com/2021/12/01/world/africa/coranavirus-vaccine-hesitancy-africa.html">Vaccine hesitancy</a>, often fueled by mis- and disinformation, inhibits vaccine uptake in all countries; yet, in countries reliant on donor funding, it is sometimes <a href="https://www.youtube.com/watch?v=qA7oos7sU8Y&amp;t=3820s">erroneously cited</a> as a cause for low coverage, rather than a challenge to be tackled with robust resources and coordinated, well-informed programs.</p>
<p>When low- and lower-middle-income countries lack access to the resources they need for vaccination programs, they are at high risk for Covid-19 surges. Furthermore, the <a href="https://data.undp.org/vaccine-equity/affordability/">finances</a> and underlying infrastructure to respond effectively to the disease are already severely constrained. Vital resources, like well-staffed and well-equipped <a href="https://www.afro.who.int/news/vaccine-supply-crunch-adds-risk-covid-19-resurgence">intensive care units, oxygen</a> supplies, and personal protective equipment (PPE) for health care workers, are all stretched to their limits. This taxes already-overburdened health systems, diverting staff and stockpiles away from other essential programs, resulting in <a href="https://www.cgdev.org/publication/indirect-health-effects-covid-19-emerging-findings-kenya-philippines-south-africa-and">negative downstream health</a> consequences. Longstanding services for infectious diseases like <a href="https://www.nature.com/articles/d41586-021-03546-8">tuberculosis, malaria, and HIV</a>, as well as <a href="https://www.cfr.org/article/noncommunicable-diseases-kill-slowly-normal-times-and-quickly-covid-19-times">noncommunicable diseases</a> like cancer and diabetes, have been severely disrupted during Covid-19. Well-established childhood immunization campaigns—one of the defining successes of global health—have suffered, with an <a href="https://www.cdc.gov/mmwr/volumes/70/wr/mm7045a1.htm?s_cid=mm7045a1_w">estimated 22 million children worldwide</a> missing their first measles vaccine in 2020, the largest drop in 20 years. Fueled by limited access to vaccines and a lack of programs that support uptake, the pandemic in low- and lower-middle income countries can become a vicious cycle. Covid-19 cases surge, straining already overburdened health systems and exacerbating conditions that may allow Covid-19 infections and deaths to increase further.</p>
<p>With largely unvaccinated populations, and large numbers of inhabitants living in <a href="https://pandem-ic.com/vaccine-inequity-the-ignominy-of-our-times/">extreme poverty</a>, low- and middle-income countries are vulnerable to frequent Covid-19 outbreaks and rapid and widespread circulation of virus. This sets the stage for the emergence of <a href="https://www.rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/">viral variants</a>. Variants can also arise in <a href="https://www.nejm.org/doi/full/10.1056/NEJMsb2104756">immunocompromised individuals</a> who develop chronic SARS-CoV-2 infections, which may be more likely when they do not receive appropriate treatment because of disruptions in health programs. When variants emerge anywhere, the global risk escalates for all people, regardless of income or geography. <strong>Local emergence of variants like Alpha and Delta quickly evolved into a global problem. Omicron may do the same.</strong></p>
<p>Detection and tracking of variants requires <a href="https://www.rockefellerfoundation.org/blog/genomic-surveillance-is-essential-to-track-covid-19-variants-in-both-unvaccinated-and-vaccinated-populations/">genomic sequencing</a>, that is, determining the genetic code of the entire virus. Sequencing is more complex than diagnostic testing, and requires specialized equipment and trained personnel.** Scientists have estimated that effective genomic surveillance requires sequencing <a href="https://www.medrxiv.org/content/10.1101/2021.01.12.21249613v1.full">5%</a> of cases at a minimum and <a href="https://magazine.jhsph.edu/2021/covid-19-how-track-viral-variants">25-30%</a> of cases for ideal surveillance coverage. Many countries lack the ability to perform that kind of surveillance.** In fact, 41 mostly low- and lower-middle-income countries have not submitted any sequences to GISAID in the past 180 days based on an analysis of <a href="https://www.gisaid.org/index.php?id=208">GISAID</a> and <a href="https://blogs.worldbank.org/opendata/new-world-bank-country-classifications-income-level-2021-2022">World Bank</a> data.</p>
</section>
<section id="sequencing-capacity" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="sequencing-capacity">Sequencing capacity</h4>
<div id="fig-2" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<p><img src="ptc2-viz2-final-1440@2x-2048x1128.png" class="img-fluid figure-img" alt="3 bar charts showing sequences shared, % of cases sequenced and shared, and median days to deposition of data per country, colored by country income level."></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: Countries that have published more than 2000 sequences to GISAID in the last 180 days</figcaption><p></p>
</figure>
</div>
<p>Sequencing capacity varies greatly based on income level, what % of cases are sequenced, how many sequences are shared, and submission latency. Of these, high income countries are alone in the capacity to sequence more than 5% of cases. It is important to note that political constraints and legitimate concerns around the benefits of sharing data may also affect the percentage of cases sequenced, beyond strict capacity limits.]</p>
<p>Variants emerging in areas lacking both vaccination and genomic surveillance may have the chance to become widespread before they can be detected and contained. The <a href="https://www.rockefellerfoundation.org/blog/making-sense-of-variants-they-all-deserve-attention-a-few-merit-concern/">Delta variant</a>, currently the dominant strain worldwide, was first identified in late 2020 in India, but was not recognized as a public health threat until March 2021, in part because India was sequencing less than 1% of positive samples. It is now the <a href="https://outbreak.info/situation-reports/delta?loc=IND&amp;loc=GBR&amp;loc=USA&amp;selected#longitudinal">dominant variant globally</a>.</p>
<p><strong>In sharp contrast to the timeline for identifying Delta, the identification of a new Variant of Concern, <a href="https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern">dubbed Omicron by the WHO</a>, happened swiftly. This illustrates the importance of effective genomic surveillance and rapid response to potentially significant variants. </strong></p>
<p>Omicron <a href="https://www.science.org/content/article/patience-crucial-why-we-won-t-know-weeks-how-dangerous-omicron">first came to light</a> when some <a href="https://www.genome.gov/about-genomics/fact-sheets/Understanding-COVID-19-PCR-Testing">PCR tests</a> in laboratories in <a href="https://www.nature.com/articles/d41586-021-03552-w">South Africa and Botswana</a> gave an unusual result called <strong><a href="https://www.bbc.com/news/health-59460252">S Gene Dropout or S-Gene Target Failure (SGTF)</a></strong>. <a href="https://www.businesswire.com/news/home/20210108005522/en/U.S.-FDA-Lists-Applied-DNA%E2%80%99s-Linea%E2%84%A2-COVID-19-Assay-Kit-as-1-of-Only-2-EUA-Authorized-Tests-Able-to-Detect-S-Gene-Mutation-Found-in-SARS-CoV-2-U.K.-Variant-B.1.1.7">Some PCR test kits</a> are designed to detect parts of three different genes in SARS-CoV-2. The virus samples these labs were testing were missing a part of the spike gene detected by this test, so it gave a negative result on one of the three parts of the test. When these S-gene dropout results were reported, researchers at the <a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(20)30116-6/fulltext">Network for Genomic Surveillance in South Africa (NGS-SA)</a> had been investigating an uptick in cases in the Gauteng province of South Africa. They found the same S-gene dropout in <a href="https://www.krisp.org.za/manuscripts/25Nov2021_B.1.1.529_Media.pdf">nearly all of the Gauteng samples</a>.</p>
<p><strong>While <a href="https://twitter.com/PPI_Insights/status/1465789602632019973?s=20">S-gene dropout can act as a proxy,</a> it is not a definitive identification of Omicron.</strong> Other variants also possess this spike gene deletion, including the Alpha variant (which has decreased markedly since its <a href="https://outbreak.info/situation-reports/alpha?loc=GBR&amp;loc=USA&amp;loc=USA_US-CA&amp;selected=GBR#longitudinal">April 2021 global peak</a>, but is still circulating—as indicated by submissions to <a href="https://www.gisaid.org/hcov19-variants/">GISAID</a>.) In addition, a <a href="https://github.com/cov-lineages/pango-designation/issues/361">sublineage of Omicron</a> has recently been identified that lacks this particular gene deletion, and thus cannot be detected by the S-gene dropout. Sequencing of the entire viral genome is still needed. When the samples that gave the unusual result were sequenced, they showed an exceedingly large number of mutations, many of them known to affect transmissibility and immune evasion. Nearly all of the samples from Gauteng gave similar results. This <a href="https://www.npr.org/2021/11/30/1060185915/the-omicron-variant-might-have-originated-in-someone-with-a-suppressed-immune-sy">new variant was different</a> from any variant previously sequenced.</p>
<p>The variant, assigned to Pango lineage B.1.1.529, was detected the afternoon of November 23 and at a <a href="https://www.youtube.com/watch?v=b4aTIoZDQk4&amp;t=3s">press briefing</a> <a href="https://www.krisp.org.za/manuscripts/25Nov2021_B.1.1.529_Media.pdf">36 hours later</a>, <a href="https://www.krisp.org.za/people.php?fullName=de%20Oliveira%20T">Professor Tulio de Oliveira</a>, director of the <a href="https://www.krisp.org.za/index.php">KwaZulu-Natal Research Innovation and Sequencing Platform (KRISP)</a> announced the new variant to the world on behalf of NGS-SA. (Professor de Oliveira also identified the <a href="https://www.afro.who.int/news/why-genomic-sequencing-crucial-covid-19-response">Beta variant</a>.) The next day, November 26, the <a href="https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern">WHO designated it as a Variant of Concern</a> and named it Omicron.</p>
<p><strong>Given this early alert, scientists immediately began studying Omicron to determine how the mutations affect the way Omicron spreads, causes disease, and responds to vaccines and treatments</strong>. The results of some of these epidemiological and laboratory studies take several weeks to yield definitive results. In the meantime, the publicly shared information and genetic sequences allow researchers and public health officials around the world to rapidly identify Omicron cases, understand the chronology and geography of its spread, and accelerate our global response. While it is too early to know what the ultimate ramifications of Omicron will be with regard to transmissibility, virulence, or immune evasion, as of mid-December 2021, the wave largely attributed to Delta continues to cause a daily average of over <a href="https://www.nytimes.com/interactive/2021/world/covid-cases.html">625,000 cases and 7,200 deaths</a> worldwide.</p>
</section>
<section id="omicron-sequences" class="level4 page-columns page-full">
<h4 class="anchored" data-anchor-id="omicron-sequences">Omicron sequences</h4>
<div id="fig-3" class="quarto-figure quarto-figure-center column-page-right anchored">
<figure class="figure">
<p><img src="ptc2-viz3-final-1440@2x-2048x1128.png" class="img-fluid figure-img" alt="Line and dot chart showing the earliest sample collection date and number of Omicron sequences shared for all 64 countries sharing Omicron sequences to GISAID from 11/5/21 to 12/16/21."></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;3: Visualizing the timeline of Omicron sequences collected and submitted to GISAID by country, with circle sizes representing number of Omicron samples collected per date, the line spanning from earliest collected sample to latest submitted sequence, color denoting continent, and ordered top-down by earliest date of collection</figcaption><p></p>
</figure>
</div>
<p>The excellent work of South African scientists and <a href="https://twitter.com/macroliter/status/1464031109428121620?s=20">other contributing researchers</a> shows how effective genomic surveillance can be at identifying and tracking variants and enabling a rapid response. The value of genomic surveillance can be further enhanced by expanding it to underserved populations, both <a href="https://www.rockefellerfoundation.org/blog/genomic-surveillance-is-essential-to-track-covid-19-variants-in-both-unvaccinated-and-vaccinated-populations/">domestically</a> and <a href="https://www.vox.com/22807542/omicron-coronavirus-variant-covid-19-california-testing-south-africa">globally</a>. The swift announcement of Omicron was a benefit to the global community, yet it was met with travel bans against African nations, without comparable bans on other countries with documented Omicron cases. These bans have been <a href="https://www.nature.com/articles/d41586-021-03608-x">criticized by scientists</a> as being ineffective, inhibiting the very transparency that had alerted the world to the variant, and interfering with the ability of researchers to study it. Inequitable application of policies is yet another facet of the global imbalance that fuels pandemics.</p>
<p><strong>In spite of these persistent issues, a steady expansion of genomic surveillance capabilities around the world will aid global surveillance efforts</strong>. The <a href="https://africacdc.org/institutes/ipg/">Africa CDC’s Institute of Pathogen Genomics</a> is playing a crucial role in coordinating and supporting expansion of capabilities and data sharing, and comparable programs have been initiated to improve genomic surveillance in other low- and lower-middle-income regions. The <a href="https://www.rockefellerfoundation.org/news/the-rockefeller-foundation-awards-new-grants-to-scale-up-covid-19-testing-in-india/">Rockefeller Foundation</a>, through its Pandemic Prevention Institute, is supporting initiatives in <a href="https://www.newindianexpress.com/cities/hyderabad/2021/dec/04/ccmb-leads-consortium-sequencing-for-omicron-2391578.html">India</a>, at the University of Ghana’s <a href="https://www.rockefellerfoundation.org/grant/university-of-ghana-2021-4/">West African Centre for Cell Biology of Infectious Pathogens</a>, and at the Nigeria-based <a href="https://massdesigngroup.org/work/design/african-centre-excellence-genomics-infectious-diseases">Africa Centre for Excellence in Genomics of Infectious Disease</a>. The Pandemic Prevention Institute is also supporting the work of <a href="https://ceri.africa/">The Centre for Epidemic Response and Innovation (CERI)</a>, a consortium of public health officials, scientists and clinicians associated with the South African Medical Research Council (SAMRC), the Department of Science and Innovation (DSI), Stellenbosch University and the University of KwaZulu-Natal (UKZN). Headed by <a href="https://ceri.africa/person/?fullName=de%20Oliveira%20T">Professor de Oliveira</a>, CERI has the <a href="https://www.rockefellerfoundation.org/news/the-rockefeller-foundation-supports-the-launch-of-the-centre-for-epidemic-response-and-innovation-to-help-prevent-future-pandemics/">goal</a> of improving genomic surveillance on the African continent. <strong>In addition to its own sequencing technology, CERI’s <a href="https://genomics.africa/">Genomics Africa Fellowship Program</a> is training scientists from countries throughout Africa, who can then support genomic surveillance in their own countries</strong>.</p>
<p>This expanded capability will help support the ability to track Covid-19 and guide responses in some regions, and additional public and private investment in surveillance initiatives are still needed to provide effective global surveillance. Genomic surveillance alone will not, however, solve the problem of continued spread caused by low vaccination rates and resulting high transmission rates. Scaling up genomic surveillance and swift data sharing depends on countries and communities seeing the benefits of sharing information—including access to the resources needed to respond. <strong>The emergence of the Omicron variant has illustrated to the world the need for vaccine equity and global genomic surveillance.</strong> Programs that expand sequencing capacity will aid in attaining comprehensive surveillance. Controlling the pandemic depends on using genomic data to track the virus, as well as taking the steps needed to end the inequities that contribute to the emergence of variants that may threaten the globe.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>